neuralpredictors/layers/readouts.py

```diff
@@ -956,6 +958,19 @@ def forward(self, x, sample=None, shift=None, out_idx=None):
                            if sample is True/False, overrides the model_state (i.e training or eval) and does as instructed
             shift (bool): shifts the location of the grid (from eye-tracking data)
             out_idx (bool): index of neurons to be predicted
+            multiplex (bool): if True, the neurons do not readout from their grid position,
+                            but from every position, i.e. every pixel, instead. Thus, each neuron is effectively copied
+                            to all positions in the image. The neuronal activity output matrix is then no longer
+                            (batch, neurons), but (batch, neurons x locations), when  the `collapse` argument is True,
+                            or (batch, neurons, height x width)
+            crop_edge_px (int): When multiplex is True, the response to all pixels that are cropped. Cropping
+                            will be symmetric by n pixels as specified by this argument.
+            collapse (bool): This argument can be used to modify the output shape of the readout when multiplex is
+                            set to True.
+                            By default (collapse=True), the readout returns the output in the shape of
+                            (batch, neurons x locations). When collapse=False, the returned shape will be:
+                            (batch, neurons, height x width)
+
 
         Returns:
             y: neuronal activity
```

Comment:
Since multiplex and collapse modify what is returned it would be good to also have that mentioned in the return docstring.

Maybe something like this:
```suggestion
            y: neuronal activity. shape: default (batch, neurons), if multiplex=True and collapse=False (batch, neurons x locations), if multiplex=True and collapse=True (batch, neurons, height x width)
```